src/applications/disability-benefits/all-claims/pages/secondaryOtherSources.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../content/ptsdClassification';
+import {
+  otherSourcesDescription,
+  otherSourcesHelpText,
+} from '../content/secondaryOtherSources';
+
+export const uiSchema = index => ({
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781a" />
+  ),
+  'ui:description': otherSourcesDescription,
+  [`incident${index}`]: {
+    otherSources: {
+      'ui:title': ' ',
+      'ui:widget': 'radio',
+      'ui:options': {
+        labels: {
+          yes: 'Yes, I’d like help getting supporting evidence and information',
+          no: 'No, I don’t need help with this.',
+        },
+      },
+    },
+  },
+  'view:otherSourcesHelp': {
+    'ui:description': otherSourcesHelpText,
+  },
+});
+
+export const schema = index => ({
+  type: 'object',
+  properties: {
+    [`incident${index}`]: {
+      type: 'object',
+      properties: {
+        otherSources: {
+          type: 'string',
+          enum: ['yes', 'no'],
```

Comment:
I think you can do the same thing here with a boolean by setting `'ui:widget': 'yesNo'` in the `uiSchema`.

Reply:
Hey @cvalarida! It seems that using the yesNo widget takes away the ability to have custom labels. Those are pretty essential for this page.

Reply:
Ahh, I see what I did wrong now. Updated @cvalarida!